modules/ROOT/pages/update-studio.adoc

```diff
@@ -0,0 +1,62 @@
+= Updating Bonita Studio
```

Comment:
⚠️ missing alias from upgrade-studio.adoc

Reply:
This page did not exist in 2021.1 or previous versions. Why do we need an alias?